maskit/masks.py

```diff
@@ -319,6 +332,11 @@ def copy(self) -> "MaskedCircuit":
         clone.default_value = self.default_value
         return clone
 
+    def temporary_full_parameters(self, changed_parameters: np.ndarray) -> np.ndarray:
```

Comment:
Not sure about the naming here; I know it's intended as ``temporary (full parameters)``, but I keep reading it as ``(temporary full) parameters`` before realising my mistake and backtracking. Perhaps something like ``expanded_parameters`` works better?